src/Containers/ClickableButton/ClickableButton.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export interface ClickableButtonProps extends React.HTMLAttributes<HTMLDivElement>{
+    buttontext: string;
+};
+
+export const ClickableButton: React.FC<ClickableButtonProps> = ({
+    buttontext, //the text that is displayed on the button.
+    onClick, //onclick not currently used but would be if the button did anything and would run on press.
+    ...props
+}): React.ReactElement => <ClickableButtonBox {...props}>{
+    <p>
+        {buttontext}
+    </p>
+}</ClickableButtonBox>;
+
+const ClickableButtonBox = styled.div`
+    ${({theme, ...props}):string => `
+        p{
+            text-align: center;
+            line-height: 30px;
+            width: 100px;
+            height: 35px;
+            overflow: hidden;
+            font-family: ${theme.font.family};
+            border-radius:${theme.dimensions.radius};
+            background-color: ${theme.colors.background};
+            color: ${theme.colors.text};
+        }
+        p:hover{
+            background-color: ${theme.colors.chairTableEditBackground};
+            color: ${theme.colors.text};
+        }
+        p:active{
+            background-color: ${theme.colors.clickedBackground};
+            color: ${theme.colors.clickedText};
+        }
+    `}
+`
+export default ClickableButton;
+
+//todo change from css to prop state for hover
```

Comment:
Waiting for todo

Reply:
in the meeting you mentioned that I should use the button in the react ui library as a guideline, but after looking at it, it is using css to change the button color as well, so im assuming that is not what you want?

Reply:
You can use the &:hover CSS if the button in react-ui is using it. but don't use it for onclick